dynamics-nav/COPYSTREAM-Function.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "COPYSTREAM Function"
+description: "Copies the information that is contained in an InStream to an OutStream."
```

Comment:
Too short per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main